doto-frontend/src/components/ModalContent.js

```diff
@@ -34,13 +34,13 @@ const useStyles = makeStyles(theme => ({
 const ModalContent = props => {
     const classes = useStyles();
 
-    const [selectedName, setSelectedName] = React.useState("TASK - " + new Date());
-    const [selectedDescription, setSelectedDescription] = React.useState("Cry in anguish");
-    const [selectedDueDate, setSelectedDueDate] = React.useState(new Date());
-    const [selectedDuration, setSelectedDuration] = React.useState(0);
-    const [selectedLocation, setSelectedLocation] = React.useState("");
-    const [selectedPriority, setSelectedPriority] = React.useState("");
-    const [selectedReminder, setSelectedReminder] = React.useState("");
+    const [selectedName, setSelectedName] = useState("TASK - " + new Date());
+    const [selectedDescription, setSelectedDescription] = useState("Cry in anguish");
```

Comment:
Do we want to keep the "cry in anguish" here

Reply:
Yes, it is important. That's how I felt while coding this.